jvb/src/test/kotlin/org/jitsi/videobridge/cc/allocation/BitrateControllerTest.kt

```diff
@@ -64,8 +64,15 @@ class BitrateControllerTest : ShouldSpec() {
      */
     override suspend fun beforeSpec(spec: Spec) = super.beforeSpec(spec).also {
         setNewConfig(
-            "videobridge.cc.bwe-change-threshold=0" +
-                "\n" + configWithMultiStreamDisabled, // Also disable multi stream support,
+            """
```

Comment:
And I'm wondering if we should have a FakeExecutorService-based test of the periodic update logic.